applications/nrf_desktop/README.rst

```diff
@@ -377,41 +376,63 @@ Check :ref:`nrf_desktop_porting_guide` for details.
 nRF Desktop build types
 =======================
 
-The nRF Desktop does not use a single :file:`prj.conf` file.
-Configuration files are provided for different build types for each supported board.
-
-Each board has its own :file:`prj.conf` file, which represents a ``debug`` build type.
-Other build types are covered by dedicated files with the build type added as a suffix to the ``prj`` part, as per the following list.
-For example, the ``release`` build type file name is :file:`prj_release.conf`.
-If a board has other configuration files, for example associated with partition layout or child image configuration, these follow the same pattern.
-
-.. include:: /config_and_build/modifying.rst
-   :start-after: build_types_overview_start
-   :end-before: build_types_overview_end
+The nRF Desktop application does not use a single :file:`prj.conf` file.
+Before you start testing the application, you can select one of the build types supported by the application, depending on your development kit and the building method.
+Not every board supports both mentioned build types.
+
+See :ref:`app_build_additions_build_types` and :ref:`modifying_build_types` for more information about this feature of the |NCS|.
+
+The application supports the following build types:
+
+.. list-table:: nRF Desktop build types
+   :widths: auto
+   :header-rows: 1
+
+   * - Build type
+     - File name
+     - Supported board
+     - Description
+   * - Debug
+     - :file:`prj.conf`
+     - All from `Requirements`_
+     - Debug version of the application; the same as the release build type, but with debug options enabled.
+   * - Release
+     - :file:`prj_release.conf`
+     - All from `Requirements`_
+     - Release version of the application with no debugging features.
+   * - Debug Fast Pair
+     - :file:`prj_fast_pair.conf`
+     - ``nrf52840dk_nrf52840``, ``nrf52840gmouse_nrf52840``
+     - Debug version of the application with `Fast Pair`_ support.
+   * - Release Fast Pair
+     - :file:`prj_release_fast_pair.conf`
+     - ``nrf52kbd_nrf52832``, ``nrf52840gmouse_nrf52840``
+     - Release version of the application with `Fast Pair`_ support.
+   * - Dongle
+     - :file:`prj_dongle.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that lets you generate the application with the dongle role.
+   * - Keyboard
+     - :file:`prj_keyboard.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that lets you generate the application with the keyboard role.
+   * - MCUboot QSPI
+     - :file:`prj_mcuboot_qspi.conf`
+     - ``nrf52840dk_nrf52840``
+     - Debug version of the application that uses MCUboot with the secondary slot in the external QSPI FLASH.
+   * - MCUboot SMP
+     - :file:`prj_mcuboot_smp.conf`
+     - ``nrf52840dk_nrf52840``
```

Comment:
Also supported by `nrf52840gmouse_nrf52840`

Reply:
Should we also mention `prj_3bleconn.conf`, `prj_4llpmconn.conf`, `prj_release_4llpmconn.conf` etc. ? (see the `sample.yaml` file of nRF Desktop application for a complete list that is up to date)

Reply:
Maintaining list in both places introduces quite serious risk of misalignment here. Could we let user refer to the `sample.yaml` file or generate the list automatically relying on `sample.yaml` file content ?

Reply:
Yes, I think the risk is considerable. What would be the standard "must-have" (or rather, "must-know") build types that we must mention here?

Reply:
Maybe `prj.conf` and `prj_release.conf` (as the most basic ones. I would also suggest to link to the `sample.yaml` content if user wants to know about other build types.